repo/meta/schema/package-schema.json

```diff
@@ -114,7 +114,8 @@
           "type": "string"
         },
         "version": {
-          "type": "string"
+          "type": "string",
+          "pattern": "^[-a-zA-Z0-9.]+$"
```

Comment:
I checked that this is compatible with all current v3Package versions. Also the fact that the CI passes proves this.